File: python/q25.py
#!/usr/bin/python3

F1=1
F2=1
temp=0
index=2
while F2//(10**999)==0:
    temp = F1+F2
    F1=F2
    F2=temp
    index += 1
print(F1)
print(F2)
print(index)

File: go/q25.go
package main

import(
	"fmt"
	"github.com/daniel-willett/common-go-methods"
)

func main(){
	var F1 string = "1"
	var F2 string = "1"
	var temp string = ""
	var index int = 2
	for len(F2)<1000{
		temp, _ = common.Addition(F1,F2)
		F1 = F2
		F2 = temp
		index += 1
	}
	fmt.Println(index)
}
